### PR TITLE
Add map

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

File: site/src/analytics-element.ts [hunk 1]
```diff
@@ -104,7 +113,7 @@ export class AnalyticsElement extends LitElement {
       height: 100vh;
       width: 100%;
       margin: auto;
-      max-width: 1440px;
+      margin-bottom: -200px;
```
Comment (bramkragten) [inline site/src/analytics-element.ts:116]: Why is this?

File: site/src/analytics-element.ts [hunk 2]
```diff
@@ -130,6 +139,9 @@ export class AnalyticsElement extends LitElement {
       padding: 16px;
       box-sizing: border-box;
     }
+    .footer.with-map {
+      margin-top: 680px;
```
Comment (bramkragten) [inline site/src/analytics-element.ts:143]: I think we can do this better? Give the map display block?

File: site/src/analytics-map.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import svgMap from "svgmap";
+import "svgmap/dist/svgMap.min.css";
+import { customElement, LitElement, property } from "lit-element";
+import { Analytics } from "./data";
+
+const isDarkMode = matchMedia("(prefers-color-scheme: dark)").matches;
+
+@customElement("analytics-map")
+export class AnalyticsMap extends LitElement {
```
Comment (bramkragten) [inline site/src/analytics-map.ts:9]: Since this doesn't have a render, this should be an updating element

File: site/src/analytics-map.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            installations: {
+              format: "{0} Installations",
+            },
+          },
+          applyData: "installations",
+          values: countries,
+        },
+      });
+    } else {
+      oldMap!.hidden = true;
```
Comment (bramkragten) [inline site/src/analytics-map.ts:46]: Why not just remove it? We recreate it anyway

Comment (ludeeus) [reply site/src/analytics-map.ts:46]: If we remove it, it can't keep it's place

**Review by bramkragten — APPROVED** [review]

**Pull request merged**